app/templates/productDetails.html

```diff
@@ -45,29 +45,25 @@
 {% else %}
 <div class="row my-5">
     <div class="col-md-6 mb-4 mb-md-0">
-        Column
+      <div class="col-md-10">
+          <div class="images p-3">
+              <div class="text-center p-4"> <img id="main-image" img src="https://api.lorem.space/image/shoes?w=304&amp;h=225" width="450" /> </div>
```

Comment:
The `img` tag can be changed as
```suggestion
              <div class="text-center p-4"> <img id="main-image" src="{{ product.img_url }}" /> </div>
```